products/early_access_features/frontend/EarlyAccessFeature.tsx

```diff
@@ -439,7 +439,7 @@ interface PersonListProps {
     earlyAccessFeature: EarlyAccessFeatureType
 }
 
-function featureFlagEnrolmentFilter(
+function featureFlagRecordingEnrolmentFilter(
```

Comment:
The most minor of minor nits, but this uses the British spelling of "Enrolment" here, but later, we have the American spelling in the variable "featureEnrollmentKey".

Reply:
Oh yeah, 100%, it's in the handbook that we should always use the American spelling, I'll fix it :)

I learned British English in school, and I'll admit I didn't even know enrollment had two spellings 😆

Reply:
We could split combine the difference and use three lll, `enrolllment` and now everybody is equally incorrect! 😆